test/ambiguous.jl

```diff
@@ -157,6 +157,27 @@ ambs = detect_ambiguities(Ambig5)
 # some ambiguities involving Union{} type parameters are expected, but not required
 @test !isempty(detect_ambiguities(Core, Base; imported=true, ambiguous_bottom=true))
 
+module AmbigStdlib
+using Test
+
+for lib in Base._stdlibs
+    @eval import $lib
+end
+
+modules = @eval [$(Base._stdlibs...)]
+
+import InteractiveUtils
+push!(modules, InteractiveUtils)
+
+# At the moment, there are some method ambiguities in stdlibs.  We
+# impose a bound to the number of detected ambiguities to avoid
+# introducing more ambiguities.
+# https://github.com/JuliaLang/julia/issues/27408
+@test length(detect_ambiguities(modules...; imported=true, recursive=true)) ≤ 2
+
+@test_broken detect_ambiguities(modules...; imported=true, recursive=true) == []
```

Comment:
Perhaps `isempty`?

Reply:
I followed the example for `Base` and `Core` as I thought it makes sense:

https://github.com/JuliaLang/julia/blob/f31e28a50e029942d5d96977f5993defb2f60c13/test/ambiguous.jl#L154-L156

Shall I add a comment to clarify it?

Reply:
No need for a comment; it's a matter of style really :). I'm a fan of using evocatively named predicate functions where they exist. Best!